controller/src/main/java/io/pravega/controller/store/stream/ZKStreamMetadataStore.java

```diff
@@ -228,7 +150,7 @@ ZKScope newScope(final String scopeName) {
 
     @Override
     public CompletableFuture<List<String>> listScopes() {
-        return storeHelper.listScopes();
+        return storeHelper.getChildren(SCOPE_ROOT_PATH);
```

Comment:
I'm not sure this is a good change. `getChildren` is leaking information about the underlying implementation, while `listScopes` leaves it abstract.

Reply:
actually this is store specific. This is in ZkStreamMetadataStore. The storeHelper is zkStoreHelper.
Note: StoreHelper is not an interface. 
ZkStore makes use of a zk helper utility for making calls to zookeeper. That utility is called ZkStoreHelper.  
It is intended to be a helper class that exposes generic `zk` calls like getChildren. 

However, it had this unnecessary `listScopes` method. So I have just removed listScopes from there and made the getChildren call here.